### ISSUE TITLE
Remove anchor from active list item

### ISSUE THREAD

**Issue by danielmrey** [wp-bootstrap/wp-bootstrap-navwalker#280]
Can the text inside an active list item be only text and not an anchor with a link?

When you are in a page the link in the menu to that page should be only text and not anchor

Thank you!

**Comment by pattonwebz** [comment]
Hey Daniel,

I agree this might be the preferred behaviour. I assigned myself to this issue and will investigate when I have time. Unfortunately I'm pretty busy this month but will get back to this as soon as I can.

Thanks for the suggestion!

**Comment by danielmrey** [comment]
Thanks to you William!

Best Regards.


*Daniel Muñoz Rey*
Freelance Web designer and developer

**Comment by szepeviktor** [comment]
WP core makes the current page's menu item a link.

**Comment by IanDelMar** [comment]
According to https://www.w3.org/WAI/tutorials/menus/structure/#indicate-the-current-item both approaches are valid. 
> This technique is particularly useful when the anchor (<a>) cannot be removed from the HTML.

may be read as if the first approach is preferable.
